internal/transformers/importelision_test.go

```diff
@@ -22,6 +23,42 @@ type fakeProgram struct {
 	getResolvedModule           func(currentSourceFile *ast.SourceFile, moduleReference string) *ast.SourceFile
 }
 
+func (p *fakeProgram) FileExists(path string) bool {
```

Comment:
Yeah, having to have all of these is sort of a shame but maybe there's no way around it ☹️

Reply:
Yup. I certainly don't wanna leak all the stuff on `program`'s internal bits into the `checker` scope - minimally it'd make writing the interfaces incredibly painful.